msat/src/msat_solver.cpp

```diff
@@ -205,31 +212,16 @@ Result MsatSolver::check_sat()
 Result MsatSolver::check_sat_assuming(const TermVec & assumptions)
```

Comment:
isnt it duplicate? we have an inline function for this in the header file. Or am I missing something?

Reply:
oh, i see now, the argument is different

Reply:
how can we make it more clear?

Reply:
Yeah good point, we could give it a different name? Like -- `msat_check_sat_assuming` or something?

Reply:
maybe give the other one the name : check_sat_assuming_msatvec?

Reply:
Cool, done with 948f7dcd7c89f478ff44fb847d7c8db7b1b430d5